### PR TITLE
Use Task API for cloning project instead of timing out in Uri Handler

### PR DESCRIPTION
**Pull request by mxsdev**

### CODE REVIEW

**Comment by christian-bromann** [comment]
@mxsdev let me know if you need any support getting this over the line.

**Comment by mxsdev** [comment]
@christian-bromann 

Sure, I think I just need to fix tests. Not a super high priority which is why it's been sitting in draft so long.

File: src/extension/handler/uri.ts [hunk 1]
```diff
@@ -111,23 +113,53 @@ export class RunmeUriHandler implements UriHandler {
...
+          }
+
+          /**
+           * only close terminal if execution passed and desired by user
+           */
+          if (e.exitCode === 0) {
+            resolve(true)
+          } else {
+            resolve(false)
+          }
```
Comment (christian-bromann) [inline src/extension/handler/uri.ts:145]:
```suggestion
          return resolve(e.exitCode === 0)
```

**Review by christian-bromann — APPROVED** [review]
Works great 🎉  tested it locally and on Windows 👍

**Pull request merged**